README.md

```diff
@@ -2,6 +2,8 @@
 
 [![made-with-python](https://img.shields.io/badge/Made%20with-Python-1f425f.svg)](https://www.python.org/)
 
+![Hello World Deploy](https://github.com/communitiesuk/funding-service-design-frontend/actions/workflows/govcloud.yml/badge.svg)
```

Comment:
This is going to stop being hello-world next sprint - probably call it the funding-service-design-frontend ?

Reply:
sure